src/modules/io.weldr.anaconda

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python3
+
+from contextlib import contextmanager
+from types import SimpleNamespace
+import json
+import os
+import subprocess
+import sys
+import tempfile
+
+options = SimpleNamespace(**json.loads(sys.stdin.read()))
+
+with tempfile.NamedTemporaryFile('w', suffix='.ks') as f:
+    f.write(options.kickstart)
+    f.flush()
+
+    cmd = [
+        'anaconda',
+        '-d', '--cmdline',
+        '--skippackages',
+        '--kickstart', f.name,
+        '--dirinstall', options.tree
+    ]
+
+    subprocess.run(cmd, check=True)
```

Comment:
(I know this is a prototype so take theses comments in that context).
We should be using pylorax tools to do this. If novirt_install doesn't provide the right pieces, break it up into ones that do. Not running anaconda directly from here.